src/scripts/inclusion-bot.yaml

```diff
@@ -209,23 +265,33 @@ triggers:
     alternatives:
       - swindled
       - cheated
+    why: >
+      ":TRIGGER:" is an abbreviated form of “gypsy,” and is based on a racist
```

Comment:
my understanding is that gypsy is racist, too. should we include that? (and/or include it in the inclusion bot?)

Reply:
I think that makes sense! Let's run it by the diversity guild, since the guild is responsible for the content here. You can add it to [the doc](https://docs.google.com/document/d/1iQT7Gy0iQa7sopBP0vB3CZ56GhyYrDNUzLdoWOowSHs/edit) and tag Jessyka, Aileen, or Ashley.

Reply:
added! thanks